src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,89 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+import { getMovieByTitle } from '../../api/api';
+import { MovieCard } from '../MovieCard';
+
 import './FindMovie.scss';
 
-// import { MovieCard } from '../MovieCard';
+type Props = {
+  onMovieAdd: (movie: Movie) => void
+};
+
+export const FindMovie: React.FC<Props> = ({ onMovieAdd }) => {
+  const [title, setTitle] = useState('');
+  const [movie, setMovie] = useState<Movie>();
+  const [isError, setIsError] = useState(false);
+
+  const onTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setIsError(false);
+  };
+
+  const onFindClick = () => {
+    getMovieByTitle(title)
+      .then(mov => setMovie(mov))
```

Comment:
do not use shorten words except wide used

```suggestion
      .then(movie => setMovie(movie))
```